src/Header/LeftNavigation.js

```diff
@@ -0,0 +1,47 @@
+import Link from "@mui/material/Link";
+
+const LeftNavigation = () => {
+  return (
+    <nav
+      style={{
+        width: "49%",
+        display: "flex",
+        alignItems: "center",
+      }}
+    >
+      <a href="/">
+        <img
+          src="https://www.pngkey.com/png/full/422-4223574_dog-and-cat-cartoon.png"
+          alt="HelloFresh logo"
+          class="logo"
+          title="Home"
+        />
+      </a>
+      <Link
+        href="cats"
+        underline="hover"
+        style={{
+          color: "whitesmoke",
+          fontSize: "26px",
+          fontFamily: "Source Sans Pro",
```

Comment:
```
color: "whitesmoke",
fontSize: "26px",
fontFamily: "Source Sans Pro",
```

you can extract this style to be `linkStyles` and can do 
```
style={{
          ...linkStyles,
          marginRight: "40px",
  }}
```

this way all the common styles for link will be in one place